src/main/java/com/iridium/iridiumteams/gui/UpgradesGUI.java

```diff
@@ -56,14 +56,21 @@ public void addContent(Inventory inventory) {
             int hours = Math.max((int) (teamEnhancement.getRemainingTime() / 3600), 0);
             String nextLevel = nextData == null ? "N/A" : String.valueOf(teamEnhancement.getLevel() + 1);
             String cost = nextData == null ? "N/A" : String.valueOf(nextData.money);
+
+            String bankCosts = "";
+            for(Double bankCost : nextData.bankCosts.values()) {
+                bankCosts = nextData == null ? "N/A" : bankCosts + " " + bankCost;
```

Comment:
What u did at here ? These is just numbers?

Reply:
yeah i havent figured out adding the name yet, im probably going to do that next

Reply:
preferably id like to add a config option to determine how that's formatted

Reply:
Just add a displayName for bank items (this also can be used on placeholders or smthing)

For now:
`StringBuilder bankCosts = new StringBuilder();
for (Map.Entry<String, Double> bankCost : nextData.bankCosts.entrySet()) {
    bankCosts.append(WordUtils.capitalizeFully(bankCost.getKey().toLowerCase())).append(": ").append(bankCost.getValue().toString());
}`